test/scripts/hexo/load_plugins.js

```diff
@@ -67,6 +64,10 @@ describe('Load plugins', () => {
 
   after(() => fs.rmdir(hexo.base_dir));
 
+  afterEach(() => {
+    createPackageFile('hexo-another-plugin');
```

Comment:
This line will reset Hexo package.json, to make sure in the following unit tests, `hexo-plugin-test` plugin is loaded from theme's package.json, not Hexo package.json